frontend/src/components/Table/Table.tsx

```diff
@@ -96,6 +96,7 @@ export interface TableProps<T extends object = {}> extends TableOptions<T> {
   detailsPanel?: DetailsOptions<T>;
   footer?: boolean;
   hideToolbar?: boolean;
+  hideBuildingFolder?: boolean;
```

Comment:
This table can be used for other list, not just for properties. Can you use a more generic name for this property

Reply:
Good suggestion.  Expanding rows should be a reusable generic implementation.

Reply:
Making a generic prop that lets you pass a function to determine specific case where to allow expansion

Reply:
yeah, I agree this should be generic if it applies to all tables or create a new component just for this use case